py/label_microservice/repo_specific_model_test.py

```diff
@@ -0,0 +1,54 @@
+"""Unittest for repo_specific_model. """
+import logging
+from unittest import mock
+import pytest
+
+from label_microservice import mlp
+from label_microservice import repo_specific_model
+from label_microservice import test_util
+
+def test_predict_labels():
+  """A unittest for predict labels.
+
+  This function mocks out the embedding service and the MLPWorker.
+  """
+  model = repo_specific_model.RepoSpecificLabelModel()
+
+  model._mlp_predictor = mock.MagicMock(spec=mlp.MLPWrapper)
+  model._mlp_predictor.predict_probabilities.return_value = [[.2, .9]]
+
+  model._label_names = ["label1", "label2"]
+  model._label_thresholds = [.5 , .5]
```

Comment:
Possible bug here?  https://github.com/kubeflow/code-intelligence/pull/82#issuecomment-569773270

Reply:
@hamelsmu apologies; I had updated the code but hadn't updated the tests. Should be fixed now. There's an open issue #75 to run the unittests automatically and report the status on the PR.